### PR TITLE
Desktop Version

### PR DESCRIPTION
**Pull request by leonardoeng13**
Portfolio: desktop version - second week, day 4.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -52,17 +52,20 @@ <h1>Leo de Andrade</h1>
...
+            </div>
+            <div class="dots">
+                <img src="images/image_dots.svg" alt="">
+            </div>
+            <div class="image-geometri1">
+                <img src="/images/image-geometry_1.svg" alt="">
+            </div>
         </div>
+
     </section>
```
Comment (Emmyn5600) [inline index.html]: - [ ] Kindly, refer to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9) to implement the following as well as fix some issues:

#### Focus on the following:

* Work on the menu links to look the same as we have on the figma like (Changing the color, Remove the underline and remove the line under the menu link)
* Add the background image as requested.
* Try to reduce the size or the max-width as well as some width to look the same as we have on the figma.
*  
    ----------------------------------Original---------------------------------------

    
    header1


    ----------------------------------Yours------------------------------------------

header2

File: index.html [hunk 2]
```diff
@@ -313,63 +317,73 @@ <h1>Get Started</h1>
...
+                        <div class="fistname">
+                            <label for="fname">First Name</label>
+                            <br>
+                            <input type="text" id="fname" name="firstname" maxlength="30" required placeholder="Your name..">
+                        </div>
+                        <div class="email">
+                            <label for="email">E-mail</label>
+                            <br>               
+                            <input type="email" id="email" name="_replyto" required placeholder="Your e-mail..">
+                        </div>
```
Comment (Emmyn5600) [inline index.html:365]: - [ ] Kindly, fix the HTML or Page Overflow-x, make it overflow-x: hidden to fix that.

File: index.html [hunk 3]
```diff
@@ -222,58 +227,57 @@ <h1 class="mobile-about-tit">About</h1>
...
+                    <div class="techprof">
+                        <div class="languages">
+                            <h3>Languages</h3>
+                            <ul>
+                                <li>JavaScript (ES5/ES6)</li>
+                                <li>Ruby</li>
+                                <li>HTML</li>
+                                <li>CSS</li>
+                            </ul>
+                        </div>
```
Comment (Emmyn5600) [inline index.html:260]: - [ ] Please refer to the figma to implement the following:

   ------------------------Original----------------------
header3


  -------------------------Yours------------------------ 
header4

File: index.html [hunk 4]
```diff
@@ -313,63 +317,73 @@ <h1>Get Started</h1>
...
-        <a href="http://stackoverflow.com"><img src="images/ic_stackoverflow.svg" alt=""></a>
-        <a href="https://github.com/leonardoeng13"><img src="images/ic_github.svg" alt=""></a>
-        <a href="https://twitter.com/andrede_leo"><img src="images/ic_twitter.svg" alt=""></a>
-        <a href="https://www.linkedin.com/in/leonardodeandrade"><img src="images/ic_linkedin.svg" alt=""></a>
-        <a href="https://medium.com/"><img src="images/ic_medium.svg" alt=""></a>
-        <a href="https://angel.co/"><img src="images/ic_angellist.svg" alt=""></a>
-    </div>
+        <form action="cv/Leonardo_Andrade_English_2021.pdf" method="get">
+            <button class="mobile-btncv" type="submit">Get my CV<img src="images/ic_link_teal.svg" alt=""></button>
+          </form>
```
Comment (Emmyn5600) [inline index.html:379]: - [ ] Check all your buttons to implement the following:
* Add hover color
* Modify your the button as cursor:pointer

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: Changes Requested :recycle:

Hi @leonardoeng13 

Good job so far!:clap::clap:
You have done well in implementing the project requirements :clap:, However, there are some issues that you still need to work on to go to the next project but you are almost there!:muscle:

##  Required Changes:writing_hand:
  * Please update the live link demo with the updated link
  
**_Check the comments under the review._**

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html (hunk 1 above)
Comment (leonardoeng13) [reply index.html]: This is a Github bug. Everything was working fine early.

Comment (Emmyn5600) [reply index.html]: Hi @leonardoeng13 sure?

File: index.html (hunk 4 above)
Comment (leonardoeng13) [reply index.html:379]: At this point, I'm using my own implementations. These requirements you're requiring changes are for next week.

File: index.html (hunk 1 above)
Comment (leonardoeng13) [reply index.html]: Fixed,

File: index.html (hunk 3 above)
Comment (Shaher-11) [reply index.html:260]: >     * [ ]  Please refer to the figma to implement the following:
>       ------------------------Original----------------------
>       header3
>       -------------------------Yours------------------------
>       header4

**Review by Shaher-11 — CHANGES_REQUESTED** [review]
#### Hello @leonardoeng13,

### Great work on making the changes requested by the previous reviewer:clap::clap::clap:
You've done a good job implementing some of the requested changes but there are still some which aren't done yet,  so we need to implement those changes to get the project approved:muscle::muscle::muscle:

 ## Required Changes ♻️

- [x] Check the comments below:point_down:

## Optional Suggestions💡

_The following point is highly recommended to improve your code quality, Please consider implementing it.

- Kindly style the nav links as shown in the original design below
#### Original design
right

#### Your Project
wrong


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Happy Coding!👏👏👏

**Review by Shaher-11 — APPROVED** [review]
#### Hello @leonardoeng13,

# Congratulations! 🎉 
#### Your project is approved!  There is nothing else to say other than... it's time to merge it :muscle: :muscle::muscle:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Cheers and Happy coding!👏👏👏

**Comment by leonardoeng13** [comment]
Version with Hoover functions.

**Pull request merged**